setup.py

```diff
@@ -42,10 +50,11 @@
                    "Topic :: Scientific/Engineering",
                    "Topic :: Scientific/Engineering :: Astronomy"],
       url="https://github.com/pytroll/pyorbital",
+      long_description=long_description,
```

Comment:
Because the readme is markdown you'll also need:

```
long_description_content_type='text/markdown',
```

Reply:
This makes it so PyPI includes the README text properly as a description of the package.

Reply:
Oh, @djhoese thanks for that! Didn't realize...